tutorials/index.md

```diff
@@ -19,43 +19,43 @@ Advanced fMRI analyses have the potential to answer questions that mainstream me
 <div class="container" markdown="1">
 ## Preview<a name="preview"></a>
 <div class="text-muted">
-The tutorials below are designed to be consumed in sequence, but are modular and can be approached in any order. We provide recommendations to scaffold on users’ prior knowledge and interest. All the notebooks are paired with data that has been preprocessed and ready for use. Our resources page has helpful links on learning the basics of fMRI pre-processing.
+The tutorials below are designed to be consumed in sequence, but are modular and can be approached in any order. We provide recommendations to scaffold on users’ knowledge and skills, though even advanced users have reported benefiting from the full sequence. All of the notebooks are paired with data that have been preprocessed and are ready for use. Our resources page has helpful links on learning the basics of fMRI preprocessing.
 </div>
 
 <div style="padding-top: 1rem;"></div>
-#### Basics
+#### Getting started
```

Comment:
If we use "Getting Started" here, we should probably change our top Getting Started link to "Install and Use"?

Reply:
I think Getting Started is fine, even if a shade overloaded (mostly about obtaining / running).

Install / use isn't quite right since colaboratory isn't install